internal/install/_static/terraform_deployer_run.sh

```diff
@@ -16,6 +16,15 @@ cleanup() {
 }
 trap cleanup EXIT INT TERM
 
+# Save GCP credentials on disk and perform authentication
+# NOTE: this is required for bq (and maybe other gcloud related tools) to authenticate
+export "GOOGLE_APPLICATION_CREDENTIALS=/root/.config/gcloud/application_default_credentials.json"
+echo "$GOOGLE_CREDENTIALS" > "$GOOGLE_APPLICATION_CREDENTIALS"
+gcloud auth login --cred-file "$GOOGLE_APPLICATION_CREDENTIALS"
```

Comment:
Is there any default location, so that `cred-file` is not needed to be defined?

Reply:
I did not find any documentation allowing to use a default location, the argument is required as is like providing username/password.

Reply:
Ok, so it looks like we need to confirm that it works as expected. It would be great if you prepare a similar test package like the special [aws](https://github.com/elastic/elastic-package/tree/main/test/packages/parallel/aws). It's the AWS integration with stripped data streams, only one left with a system test using Terraform service deployer to create the EC2 machine.

Here is my proposal on the plan:

You need to do a similar exercise with GCP. You can use this PR to push the sample package, but please make sure it's stripped and CI discovers it. Then, please you can execute the system test, but I'm afraid that it may fail due to configured environment variables. It's about modifying the Jenkinsfile to enable those envs. Then, we should observe some issues around `bq` and terraform service deployer, and eliminate them.

Reply:
> Ok, so it looks like we need to confirm that it works as expected.

May you clarify what should be confirmed? 

What's the goal for this test package? Is to test if tests are being able to be executed correctly through terraform deployer?

Reply:
Yes, that's the goal - to verify if the terraform service deployer works correctly with Google Cloud: auth, bq, Jenkins changes.

Reply:
@mtojek I added the test package, the pipeline is green but is it really running some tests? https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Felastic-package/detail/PR-662/8/pipeline/279